superset/assets/javascripts/explorev2/components/SaveModal.js

```diff
@@ -0,0 +1,249 @@
+/* eslint camel-case: 0 */
+import React, { PropTypes } from 'react';
+const $ = window.$ = require('jquery');
```

Comment:
can we use `import $ from 'jquery';` here? don't think we need `window.$` in this case.